GreenField-Org/ultracrewapi
# 8 get user

## DEV NOTE 
The changes remove the ```.env``` file from the repo.  you will need to set one up locally.  Make sure it does not get pulled into the project.

## Steps to Test 
- Get a document id from the users collection
- add it to your api call ```/api/user/ID```
- verify a 200 return
- add an unknown id to your api call ```/api/user/555884848483
- verify a 404



closes #8
